IDisposableExercise/IDisposableExercise/Person.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.Win32.SafeHandles;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IDisposableExercise
+{
+    /// <summary>
+    /// Represents a person.
+    /// </summary>
+    public class Person : IDisposable
+    {
+        /// <summary>
+        /// Represents the name of a person.
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Flag: Has Disponse already been called?
+        /// </summary>
+        public bool disposed = false;
+        // Instantiate a SafeHandle instance.
+        SafeHandle handle = new SafeFileHandle(IntPtr.Zero, true);
+
+        /// <summary>
+        /// This method is a public implementation of Dispose callable by consumers.
+        /// </summary>
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
```

Comment:
You don't have to call GC.SuppressFinalize(this) if you don't implement a finalizer on the class. And, since this class doesn't have any unmanaged resources, you don't need a finalizer. It doesn't hurt to include this call, but it is superfluous since there is no finalizer.